macosx/VDKQueue/VDKQueue.mm

```diff
@@ -0,0 +1,329 @@
+//    VDKQueue.mm
+//  Copyright © 2017-2022 Transmission authors and contributors.
```

Comment:
This feels questionable to me; surely this is a derived work? Seems to me that either Bryan D K Jones or Uli Kusterer should retain the copyright.

Reply:
They are mentioned in the next line.